recipes/R/Retail - GradientBoosting/R/applicationTrainer.R

```diff
@@ -66,6 +66,10 @@ applicationTrainer <- setRefClass("applicationTrainer",
       # Data Preparation/Feature Engineering
       #########################################
       timeframe <- configurationJSON$timeframe
+      if(any(names(df) == '_id')) {
+        utils <- Utils$new()
+        df = utils$mapFields(df)
+      }
```

Comment:
x3

Reply:
Hi @rjrussell77 
Yes, I know. But I needed to do that right after data load, and it is strange, but we have the same data load in all classes. So I had to put at 3 places.
I mentioned that earlier, we should refactor a bit samples project, but I would require some time and refactoring also at ml-runtime-r level. 
I thought for now to just update existing code for schema changes.